src/guiguts/application.py

```diff
@@ -45,21 +41,24 @@ def __init__(self) -> None:
 
         Creates windows and sets default preferences."""
 
-        self.parse_args()
+        self._parse_args()
```

Comment:
What is pylint complaining about that's causing these renames? Is it being opinionated about "private" functions  being prefaced by `_` or something?

Reply:
I'm curious about that too. My current master branch has this particular line of code at line 48 of application.py, and pylint isn't complaining about anything on that line (or very near there either) for me.

(I didn't review the overall PR - I just checked this line quickly because Casey called it out.)

Reply:
Pylint was complaining about too many public methods in the class, and those methods are only called internally in the class, so I tried adding the underscore to flag them as "protected". There are similar complaints about numbers of methods/attributes elsewhere that I didn't tackle. It would probably be good to consider if we think whether we want to do anything about that style of complaint, or if we'd rather suppress them. Any thoughts?

Reply:
Added-underscore changes reverted now. I'm happy to go with your thoughts @tangledhelix and @cpeel on a the best way to handle the "too many/too few" pylint messages

Reply:
I would disable those checks rather than doing gymnastics for the sake of appeasing it. I don't think those types of checks are going to make the code inherently better.

Reply:
I've disabled those, but left the ones about too many arguments and too many instance attributes, in case we want to decide that they are reasonable warnings that we will generally try to abide by (possibly adjusting the threshold number that causes the warning). 
If you'd rather these were not disabled for now, @tangledhelix, I don't mind reverting this last commit, merging this PR, and then having a discussion about which of the remaining warnings we want to disable/adjust/fix.

Reply:
I agree with Casey.